src/ast/ast_validations.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018-2020 Redis Labs Ltd. and Contributors
+ * Copyright 2018-2021 Redis Labs Ltd. and Contributors
```

Comment:
Side note: we should have a script which updates our Copyright comment 😄

Reply:
Yeah, I actually have a sed one-liner to accomplish this. I'll use it in January, I guess!